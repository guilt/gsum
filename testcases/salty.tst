Name: salty
Description: Test 1K file with bcrypt and custom salt
File: salty.txt 1K fill 0x42
AlgoMatrix: bcrypt-sha512 salty123
