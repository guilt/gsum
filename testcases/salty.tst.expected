Hash-bcrypt-sha512: $2a$10$Os3dJtJ2aBLFD1h84GTp8euje5YQ/iO6fN8F5oeAgBhwHK0gLldwK
